tests/toranj/openthread-core-toranj-config.h

```diff
@@ -268,6 +268,16 @@
  */
 #define OPENTHREAD_CONFIG_IP6_MAX_EXT_MCAST_ADDRS 4
 
+/**
+ * @def OPENTHREAD_CONFIG_MAC_COLLISION_AVOIDANCE_DELAY_ENABLE
+ *
+ * Define as 1 to enable collision avoidance delay feature, which adds a delay wait time after a successful frame tx
+ * to a neighbor which is expected to forward the frame. This delay is applied before the next direct frame tx (towards
+ * any neighbor) on an FTD.
+ *
+ */
+#define OPENTHREAD_CONFIG_MAC_COLLISION_AVOIDANCE_DELAY_ENABLE OPENTHREAD_FTD
```

Comment:
```suggestion
#define OPENTHREAD_CONFIG_MAC_COLLISION_AVOIDANCE_DELAY_ENABLE 1
```

nit: We normally define to 1 for FTD only feature that is on by default on FTD. 

Moreover, `OPENTHREAD_FTD` does not work in configurations because `OPENTHREAD_FTD` is not defined when building platform library.

Reply:
Ah! Good catch! I intended to change this actually (it left over from testing).Yes. I agree we should not use `OPENTHREAD_FTD` in the custom project configs for sure.
(though it may kind of work ok for `toranj` test since it is built specifically just for tesing and with simulation platform).

Reply:
Updated in new push.

Reply:
Thanks